util/linuxfw/nftables_runner.go

```diff
@@ -786,17 +795,23 @@ func insertLoopbackRule(
 
 // getNFTByAddr returns the nftables with correct IP family
 // that we will be using for the given address.
-func (n *nftablesRunner) getNFTByAddr(addr netip.Addr) *nftable {
+func (n *nftablesRunner) getNFTByAddr(addr netip.Addr) (*nftable, error) {
+	if addr.Is6() && !n.v6Available {
```

Comment:
only checking for IPv6 - there is no path via which the IPv4 table could be nil.